kyuubi-server/src/main/scala/org/apache/kyuubi/engine/flink/FlinkProcessBuilder.scala

```diff
@@ -54,27 +54,22 @@ class FlinkProcessBuilder(
     val buffer = new ArrayBuffer[String]()
     buffer += s"bash"
     buffer += s"-c"
-    val commandStr = new StringBuilder()
+    val commandSet = new LinkedHashSet[String]()
 
-    commandStr.append(s"source $FLINK_HOME${File.separator}bin" +
+    commandSet.add(s"source $FLINK_HOME${File.separator}bin" +
       s"${File.separator}config.sh && $executable")
 
     // TODO: How shall we deal with proxyUser,
     // user.name
     // kyuubi.session.user
     // or just leave it, because we can handle it at operation layer
-    commandStr.append(s" -D$KYUUBI_SESSION_USER_KEY=$proxyUser ")
+    commandSet.add(s"-D$KYUUBI_SESSION_USER_KEY=$proxyUser")
```

Comment:
why not pass via --conf

Reply:
I found that hive is passed through the -D parameter, so it remains consistent with hive. Do I also need to change hive's?

Reply:
yes

Reply:
OK I will change it later